stevePObjects/src/carPackage/carClass.java

```diff
@@ -0,0 +1,41 @@
+package carPackage;
+
+/**
+ *Car class
+ * @author Steve
+ */
+public class carClass {
```

Comment:
You should try to avoid adding class to the class name. Also class names are uppercase camelcase.